cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
+1 for changing defaults on sync when we make this change

Happy to continue discussion for timing on this. Local Rais development without 2-way sync doesn't work very well. Looking forward to wider Docker for Mac availability here.